src/collection-view.js

```diff
@@ -634,7 +634,17 @@ const CollectionView = Backbone.View.extend({
   },
 
   // Render the child's view and add it to the HTML for the collection view at a given index, based on the current sort
-  addChildView(view, index) {
+  addChildView(view, index, options = {}) {
+    if (_.isObject(index)) {
+      options = index;
+    }
+
+    let preventRender = options.preventRender;
+    //if options has defined index we should use it
+    if (options.index != null) {
```

Comment:
following conventions elsewhere this should be `index = options.index || index;`

Reply:
If options.index === 0 will fail

Reply:
ah yes good catch.  My bad.  It must be as it was.